boards/remote-reva/Makefile.include

```diff
@@ -0,0 +1,7 @@
+USEMODULE += remote-common
+
+# define the default port depending on the host OS
+PORT_LINUX  ?= /dev/ttyUSB0
```

Comment:
Doesn't this board use a dual channel FTDI as well? (Cause for prototype A you set `ttyUSB1` as a default)